data_structures/import_edge.hpp

```diff
@@ -51,6 +52,7 @@ struct NodeBasedEdge
     NodeID target;
     NodeID name_id;
     EdgeWeight weight;
+    EdgeWeight duration;
```

Comment:
The padding below allows to have a smaller struct. We wouldn’t gain anything from a memory standpoint. However from a coherence point of view do we want the same padding everywhere.